### PR TITLE
[17.0] [MIG] partner_industry_secondary

### PR DESCRIPTION
**Pull request by Tisho99**
Supersedes #1679

T-6089

### CODE REVIEW

**Comment by Tisho99** [comment]
@ValentinVinagre @HaraldPanten

File: partner_industry_secondary/models/res_partner_industry.py [hunk 1]
```diff
@@ -23,16 +23,13 @@ class ResPartnerIndustry(models.Model):
...
-                res.insert(0, cat.name)
-                cat = cat.parent_id
-            return res
-
-        return [(cat.id, " / ".join(get_names(cat))) for cat in self]
+    @api.depends("name", "parent_id")
+    def _compute_display_name(self):
+        for rec in self:
+            if rec.parent_id:
+                rec.display_name = rec.parent_id.display_name + " / " + rec.name
```
Comment (ValentinVinagre) [inline partner_industry_secondary/models/res_partner_industry.py]: change it to .format .
and in line  52/49 
`default["name"] = self.name + " 2"`

**Review by ValentinVinagre — CHANGES_REQUESTED** [review]
some changes.

**Review by ValentinVinagre — APPROVED** [review]
LGTM 👍🏻

**Review by HaraldPanten — APPROVED** [review]
Functional review. LGTM

**Pull request merged**